packages/datagateway-common/src/state/middleware/dgcommon.middleware.tsx

```diff
@@ -0,0 +1,70 @@
+import { AnyAction, Dispatch, Middleware } from 'redux';
```

Comment:
This middleware has been created and yet isn't used anywhere or has any tests. This is needed by both search and table (they might need their own in the future if they have messages to broadcast that are specific to them...) so this middleware should be deleted in `table`, the tests moved to `common` and for both `table` and `search` to add the middleware to their redux store from `common`. (the middleware is set up alongside the redux store in `App.tsx`)